keras/src/trainers/data_adapters/py_dataset_adapter.py

```diff
@@ -52,6 +58,9 @@ class PyDataset:
         over the dataset. They are not being used by the `PyDataset` class
         directly. When you are manually iterating over a `PyDataset`,
         no parallelism is applied.
+    - `shuffle=False` keeps the sample order fixed across epochs.
+        For distributed or deterministic training prefer
+        `shuffle=False` and manage the order externally.
```

Comment:
The `shuffle` argument is not part of `PyDataset.__init__`, so it should be removed from here. It's just an argument in your example.